%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Trey Hunner (http://www.treyhunner.com/)
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass{resume} % Use the custom resume.cls style

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
\usepackage[colorlinks=true, urlcolor=blue, linkcolor=red]{hyperref}

\name{Liang (Adam) Ding} % Your name
\address{511 Live Oak Ave Unit 4, Arcadia, CA 91006} % Your address
%\address{123 Pleasant Lane \\ City, State 12345} % Your secondary addess (optional)
\address{(956)~$\cdot$~802~$\cdot$~8972 \\ adamdingliang@gmail.com} % Your phone number and email

\begin{document}
%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Experience}
\begin{rSubsection}{Spatial Genomics, Inc.}{March 2024 - Present}
{Senior Bioinformatics Scientist}{Pasadena, CA}
\item Spearheaded collaboration with assay scientists to leverage spatial transcriptomics for improving cell-type identification accuracy in human solid tumors and mouse MASH liver, identifying disease-associated spatial niches and immune-tumor cell communications.

\item Led development of a novel barcode optimization algorithm using bulk and/or scRNA-seq references to resolve optical crowding in raw image decoding; documented implementation protocols for software engineers, enhancing decoding efficiency.

\item Architected customer-facing \href{https://tutorials.spatialgenomics.com/intro.html}{tutorials} for \href{https://spatialgenomics.com/product/}{GenePS} data analysis (clustering, annotation, integration, imputation, etc.) to accelerate platform adoption.

\item Translated technical insights into business collateral (brochures, posters) showcasing GenePS applications for key stakeholders.
\end{rSubsection}

\begin{rSubsection}{Spatial Genomics, Inc.}{September 2022 - February 2024}
{Bioinformatics Scientist}{Pasadena, CA}
\item Pioneered a deep generative-based (autoencoders and Bayesian inference) spatial niche alignment algorithm integrating spatial, imaging, and expression data to correct misassigned transcripts from imperfect segmentation.

\item Developed spatial data showcases (e.g., \href{http://kidneyviewer.spatialgenomics.com/}{mouse kidney Minerva story}) to demonstrate GenePS capabilities at launch.

\item Executed comparative analysis of commercial in-situ platforms to guide 2nd-generation instrument/assay R\&D.

\item Designed standardized/custom gene panels and probes for internal/external use.
\end{rSubsection}


\begin{rSubsection}{St. Jude Children's Research Hospital}{December 2018 - September 2022}
{Bioinformatics Research Scientist}{Memphis, TN}
\item Directed development of systems biology tools for multiomics analysis (scRNA-seq, spatial transcriptomics, gene networks), including:\\
\href{https://www.nature.com/articles/s41467-025-59620-6}{scMINER} (Python/R): Preprocessing, QC, clustering, gene network reconstruction, hidden driver inference, and visualization.\\
\href{https://www.nature.com/articles/s41467-023-38335-6}{NetBID2} (R): Data-driven network-based Bayesian inference of drivers.\\
\href{https://github.com/jyyulab/SJARACNe}{SJARACNe} (Python/C++): Scalable gene network reverse engineering.

\item Applied computational approaches to identify cancer biomarkers/therapeutic targets with direct clinical relevance.

\item Co-authored manuscripts and grant proposals to secure research funding.
\end{rSubsection}


\begin{rSubsection}{St. Jude Children's Research Hospital}{July 2016 - December 2018}
{Senior Software Engineer}{Memphis, TN}
\item Engineered bioinformatics workflows for clinical/research genomics (WES, WGS), improving patient data analysis pipelines.

\item Analyzed clinical sequencing data to inform treatment decisions and enhance patient care.

\item Collaborated with scientists to develop algorithms for multiomics data analysis (e.g., \href{https://github.com/stjude/RNAIndel}{RNAIndel}, \href{https://github.com/stjude/Episomizer}{Episomizer}).
\end{rSubsection}

%------------------------------------------------

\begin{rSubsection}{University of Georgia, Athens, Georgia}{September 2011 - June 2016}
{Graduate Instructor/Lab Instructor}{Athens, Georgia}
\item Designed/lectured system programming courses (C, Java); developed labs, projects, and assessments for 30+ students.
\end{rSubsection}

\end{rSection}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Education}
{\bf University of Georgia, Athens, Georgia} \hfill {\em July 2016} \\ 
Ph.D., Computer Science
%Minor in Linguistics \smallskip \\
%Member of Eta Kappa Nu \\
%Member of Upsilon Pi Epsilon \\
%Overall GPA: 5.678

{\bf University of Texas - Pan America, Edinburg, Texas} \hfill {\em August 2011} \\ 
M.S., Computer Science

{\bf Zhengzhou University, Henan, China} \hfill {\em December 2008} \\ 
M.S., Applied Mathematics\\
B.S., Applied Mathematics
\end{rSection}

%----------------------------------------------------------------------------------------
%	TECHNICAL STRENGTHS SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Technical Strengths}

\begin{tabular}{ @{} >{\bfseries}l @{\hspace{6ex}} l }
Computer Languages & Python, Rust, R, Bash, C++/C, SQL, Latex, Javascript \\
Databases & PostgreSQL, SQLite \\
Tools & PyTorch, Nvidia RAPIDS, Nextflow, Snakemake, Git, Docker \\
Single-cell/Spatial analysis & Scanpy, Seurat, scvi-tools, Squidpy, SpatialData, Minerva\\
Image analysis & ImageJ/Fiji, OpenCV, Baysor, Cellpose\\
\end{tabular}

\end{rSection}

%----------------------------------------------------------------------------------------
%	Publications
%----------------------------------------------------------------------------------------

\begin{rSection}{Publications}
* First authors contributed equally.  $^{\#}$ Corresponding authors. Sort by year in descending order.

\textbf{Peer Reviewed Journals}
\begin{enumerate}
\item Pan Q.*, \textbf{Ding L.}*, Hladyshau S., Yao X., et al., Chi H., Yu J.$^{\#}$ \href{https://github.com/jyyulab/scMINER}{scMINER}: a mutual information-based framework for clustering and hidden driver inference from single-cell transcriptomics data. Nature Communications 16, 4305, 2025.

\item McCastlain K.*, Welsh C.*, Ni Y.*, \textbf{Ding L.}*, et al., Yu J., Pounds S., Kundu M.$^{\#}$ Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells. Science Advanced, 2025.

\item Huang X.*, Li Y.*, Zhang J.*, Yan L., Zhao H., \textbf{Ding L.}, et al., Yu J.$^{\#}$, Yang J.$^{\#}$ Single-cell systems pharmacology identifies development-driven drug response and combination therapy in B cell acute lymphoblastic leukemia. Cancer cell, 42(4), 2024.

\item Yan K.*, Condori J., Ma Z., \textbf{Ding L.}, Dhungana Y., et al., Gottschalk S.$^{\#}$, Yu J.$^{\#}$ Integrome signatures of lentiviral gene therapy for SCID-X1 patients. Science Advances, 9(40), 2023.

\item Zhang Y.*, Pool A.H., Wang T., Liu L., Kang E., Zhang B., \textbf{Ding L.}, Frieda K., Palmiter R., Oka Y.$^{\#}$ Parallel neural pathways control sodium consumption and taste valence. Cell, 186(26), 2023.

\item Dong X.*, \textbf{Ding L.}, et al., Chi H., Zhang J., Yu J.$^{\#}$ \href{https://github.com/jyyulab/NetBID}{NetBID2} provides comprehensive hidden driver analysis. Cell, 14(1), 2023.

\item Chang D.*, \textbf{Ding L.}*, Malmberg R., Robinson D, Wicker M., Yan H., Martinez A., Cai L.$^{\#}$ \href{https://www.sciencedirect.com/science/article/pii/S277241582200013X}{Optimal learning of Markov $k$-tree topology}. Journal of Computational Mathematics and Data Science, 4(100046), 2022.

\item Shi H.*, Yan K.*, \textbf{Ding L.}, Qian C., Chi H., Yu J.$^{\#}$ Network Approaches for Dissecting the Immune System. iScience 23(8), 2020.

\item Silveira A.B., Kasper L.H., Fan Y., et al., \textbf{Ding L.}, Zhang J., Finkelstein D., et al., Baker S.J.$^{\#}$ H3.3 K27M depletion increases differentiation and extends latency of diffuse intrinsic pontine glioma growth in vivo. Acta Neuropathol. 137(4): 637-655, 2019.

\item Kohei K.*, \textbf{Ding L.}, Michael N.E., Stephen V.R., Scott N., John E., Juncheng D., Soheil M., Rhonda E.R., Michael R., Zhang, J.$^{\#}$ \href{https://github.com/stjude/RNAIndel}{RNAIndel}: discovering somatic coding indels from tumor RNA-Seq data. Bioinformatics, btz753, 2019.

\item Silveira A.B.*, Kasper L.H., Fan Y., Jin H., Wu G., Shaw T., Zhu X., Larson J.D., Easton J., Shao Y., Yergeau D.A., Rosencrance C., Boggs K., Rusch M.C., \textbf{Ding L.}, Zhang J., et al., Zhang J., Baker S.J.$^{\#}$ H3.3 K27M depletion increases differentiation and extends latency of diffuse intrinsic pontine glioma growth in vivo. Acta Neuropathologica. 137(4):637-655, 2019.

\item Xu K.*, \textbf{Ding L.}, Chang T.C., et al., Baker S.J., Wu G.$^{\#}$ Structure and evolution of \href{https://link.springer.com/article/10.1007/s00401-018-1912-1}{double minutes} in diagnosis and relapse brain tumors. Acta Neuropathologica. 137(1):123-137, 2019.

\item Alexander T.B.*, Gu Z.*, Iacobucci I.*, Dickerson K., Choi J.K., Xu B., et al., \textbf{Ding L}, Liu Y., Zhang J., et al., Inaba H.$^{\#}$, Mullighan C.G.$^{\#}$ The genetic basis and cell of origin of mixed phenotype acute leukaemia. Nature, 562(7727):373-379, 2018.

\item Mohebbi M.*, \textbf{Ding L.}, Malmberg R.L., Momany C., Rasheed K., Cai L.$^{\#}$ Accurate prediction of human miRNA targets via graph modeling of miRNA-target duplex. Journal of Bioinformatics and Computational Biology, 7:1850013, 2018.

\item \textbf{Ding L.}*, Xue X., LaMarca S., Mohebbi M., Samad A., Malmberg R.L.$^{\#}$, Cai L.$^{\#}$ Accurate Prediction of \href{https://academic.oup.com/bioinformatics/article/31/16/2660/321332}{RNA Nucleotide Interactions} with Backbone $k$-Tree Model. Bioinformatics, 31(16): 2660-2667, 2015.
\end{enumerate}


\textbf{Conference Proceedings}
\begin{enumerate}
\item \textbf{Ding L.}*, Xue X., LaMarca S., Mohebbi M., Samad A., Malmberg R.$^{\#}$, and Cai L.$^{\#}$ (2014) Abinitio prediction of RNA nucleotide interactions with backbone k-tree model, Proceedings of ECCB'14 Workshop on Computational Methods for Structural RNAs, Strasbourg France, 25-42.

\item \textbf{Ding L.}*, Samad A., Xue X., Huang X., Malmberg R.$^{\#}$, and Cai L.$^{\#}$ (2014) Stochastic k-tree grammar and its application in bimolecular structure modeling, International Conference on Language and Automata Theory and Applications (LATA 2014) Vol 8370, 308-322.

\item \textbf{Ding L.}*, Robertson J., Malmberg R.$^{\#}$, and Cai L.$^{\#}$ (2013) Protein closed loop prediction from contact probabilities, International Symposium on Bioinformatics Research and Applications (ISBRA 2013), 199-210.

\item \textbf{Ding L.}, Fu B., and Zhu B. (2011) Minimum Interval Cover and Its Application to Genome Sequencing. International Conference on Combinatorial Optimization and Applications (COCOA 2011), 287- 298.

\item \textbf{Ding L.}, Fu B., and Fu Y. (2010) Improved Sublinear Time Algorithm for Width- bounded Separators. International Workshop on Frontiers in Algorithmics, 101-112.

\item \textbf{Ding L.}, Fu B., Fu Y., Lu Z., and Zhao Z. (2010) $O((log n)^2)$ Time Online Approximation Schemes for Bin Packing and Subset Sum Problem. International Workshop on Frontiers in Algorithmics, 250-261.
\end{enumerate}


\textbf{Book Chapters}
\begin{enumerate}
\item Cai L., \textbf{Ding L.}, Huang X., Malmberg R.L., and Xue X. (2014) Stochastic grammar systems for biomolecular structure modeling, Integrative Bioinformatics for Biomedical Research: A No Boundary Thinking Approach, Huang and Moore ed. Cambridge Press, to appear.

\item \textbf{Ding L.}, Fu B. (2013) Algebrization and Randomization Methods. Handbook of Combinatorial Optimization, Springer, pp 171-220.

\item \textbf{Ding L.}, Fu B., Fu Y., Wan Y. (2012) Application of Width-Bounded Separators to Protein Side-Chain Packing Problem, Sequence and Genome Analysis: Methods and Applications, iConcep Press.
\end{enumerate}


\textbf{Preprints}
\begin{enumerate}
\item Michael R., \textbf{Ding L.}, Sasi A., Andrew T., Hongjian J., Michael M., Lawryn K., Andre S., Michael A.D., Suzanne J.B., Zhang J.. XenoCP: Cloud-based BAM cleansing tool for RNA and DNA from Xenograft. bioRxiv 843250.

\item \textbf{Ding L.}, Samad, A., Xue, X., Huang, X., Cai, L. (2013) Polynomial kernels collapse the W-hierarchy. arXiv preprint arXiv:1308.3613.
\end{enumerate}
\end{rSection}

%----------------------------------------------------------------------------------------

%----------------------------------------------------------------------------------------
%	HONORS and AWARDS
%----------------------------------------------------------------------------------------
\begin{rSection}{Honors and Awards}
\begin{itemize}
\item Ovation Silver Award, St. Jude Childrenâ€™s Research Hospital, 2018
\item Dissertation Completion Award, Graduate School, University of Georgia, Athens, Georgia, 2015
\item Outstanding Graduate Teaching Assistant Award,  Graduate School, University of Georgia, Athens, Georgia, 2015
\item CUDA and GPU Programming Certificate CUDA, Department of Computer Science, University of Georgia, 2013
\item Outstanding Student Scholarship, Zhengzhou University, Henan, China, 2005
\end{itemize}
\end{rSection}

\end{document}
